app/assets/javascripts/short-form/ShortFormApplicationController.js.coffee

```diff
@@ -965,17 +980,20 @@ ShortFormApplicationController = (
       if $scope.appIsSubmitted(previousApp)
         # My Applications page is now in React, prevent the "Leave Site?" popup when redirecting
         $window.removeEventListener('beforeunload', ShortFormApplicationService.onExit)
+        $window.removeEventListener('unload', $scope.onUnload)
         doubleSubmit = !! $scope.appIsSubmitted($scope.application)
         if $window.ACCOUNT_INFORMATION_PAGES_REACT is "true"
           currentUrl = window.location.origin
           newUrl = "#{currentUrl}/my-applications?"
           if previousApp.id
+            AnalyticsService.trackApplicationAbandon($scope.listing.Id, AccountService.loggedInUser?.id || null, 'Already Submitted')
             newUrl += "alreadySubmittedId=#{previousApp.id}"
             if doubleSubmit
               newUrl += "&"
           if doubleSubmit
             # As we rebuilt the My Applications page in React we were not able to figure out a way to trigger the Double Submit Modal.
             # We are leaving the code here both to document past behavior and to protect the application in case somehow the modal is triggered
+            AnalyticsService.trackApplicationAbandon($scope.listing.Id, AccountService.loggedInUser?.id || null, 'Double Submitted')
```

Comment:
This has not been triggered right? Only there to cover our bases?

Reply:
Yes! I honestly added this for me to see if it is ever being triggered, based on my exploration of the code this situation should never occur